packages/server/pipeStreamOverResponse.ts

```diff
@@ -0,0 +1,46 @@
+import {HttpResponse} from 'uWebSockets.js'
+import fs from 'fs'
+
+const pipeStreamOverResponse = (
+  res: HttpResponse,
+  readStream: fs.ReadStream,
+  totalSize: number
+) => {
+  res.onAborted(() => {
+    readStream.destroy()
+    res.done = true
+  })
+  readStream
```

Comment:
largely unused in production since our files get served from the CDN (the service worker and first HTML skeleton is the only exception).

useful for development, i suppose.